frontend/src/app/actions/varianttests.js

```diff
@@ -0,0 +1,8 @@
+
+import type { ChooseTestsAction } from '../reducers/varianttests';
+
+export function chooseTests(): ChooseTestsAction {
```

Comment:
As we don't want to synchronously run the chooseTests code, but want to wait until we are running in the browser to choose them, we now need a chooseTests action which we can trigger in the browser to update the test cohort state.